enumerables.rb

```diff
@@ -0,0 +1,135 @@
+# rubocop:disable Metrics/ModuleLength
+# rubocop:disable Metrics/PerceivedComplexity,Metrics/CyclomaticComplexity
+module Enumerable
+  def my_each
+    return to_enum(:my_each) unless block_given?
+
+    each do |num|
+      yield(num)
+    end
+  end
```

Comment:
Based on the instructions of this project you should not use any enumerable methods inside your defined methods. But I can see you are using it in your my_each. Isn't it possible to complete my-each without using each inside it? I organized the code in the following form.
```ruby
def my_each
    return to_enum(:my_each) unless block_given?

    for item in self do
      yield(item)
    end
    self
end
```